pom.xml

```diff
@@ -93,6 +97,11 @@
 			<artifactId>h2</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc10</artifactId>
+			<version>19.16.0.0</version>
```

Comment:
After taking to Om, I believe we can go with `ojdbc11` version `21.3.0.0`. Should work fine.

Reply:
Can we go with the newest version, 21.7.0.0 (https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc11)?

Reply:
Yup. I think so. That version came out from a conversation with Om. I think a newer version should work just fine.

Reply:
Since we don't specify versions for other dependencies, I took it out of the driver.